src/erros/notFoundError.js

```diff
@@ -0,0 +1,9 @@
+import baseError from "./baseError.js";
+
+class notFoundError extends baseError {
+  constructor(mensagem = "Página não encontrada") {
```

Comment:
Em se tratando de uma API que não tem nenhum recurso visual, apenas lida com dados, que tal modificar essa mensagem default para:

```suggestion
  constructor(mensagem = "Recurso não encontrado") {
```

Assim fica mais condizente com o retorno da API 😉

Reply:
✅